### PR TITLE
Allow load remote site.

### PR DESCRIPTION
**Pull request by MiyamuraMiyako**
Allow load remote site.

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -75,7 +75,11 @@ func Run(o Options) (err error) {
 
 	// Init window
 	var w *astilectron.Window
-	if w, err = a.NewWindow(filepath.Join(a.Paths().BaseDirectory(), resourcesPath, "app", o.Homepage), o.WindowOptions); err != nil {
+	var url = o.Homepage
+	if !strings.HasPrefix(o.Homepage, "http") && !strings.HasPrefix(o.Homepage, string(filepath.Separator)) {
```
Comment (asticode) [inline run.go]: I've thought about it and I'd rather you use the following condition:

```go
if strings.Index(o.Homepage, "://") == -1 && !strings.HasPrefix(o.Homepage, string(filepath.Separator)) {
```

That way one can use a file named `http.html` for instance and any kind of protocol.

**Review by asticode — CHANGES_REQUESTED** [review]
One last change.

**Comment by MiyamuraMiyako** [comment]
I updated my code.

**Review by MiyamuraMiyako — COMMENTED** [review]
To be honest, this is my first pr, I updated the code, but I do not know how to continue.

**Comment by asticode** [comment]
PR has been merged.

Cheers.

**Pull request closed without merging**